packages/qwik-city/runtime/src/route-matcher.unit.ts

```diff
@@ -163,3 +163,18 @@ regression5080('/[...rest]/suffix', () => {
   equal(matchRoute('/[...rest]/suffix', '/a/b/c/suffix/'), { rest: 'a/b/c' });
 });
 regression5080.run();
+
+const regression5126 = suite('routeMatcher/#5126');
+regression5080('/[...dynamicOne]/static-segment/[...dynamicTwo]/', () => {
```

Comment:
Should this be 5080 or 5126? @mhevery

Reply:
opps! cleanup https://github.com/BuilderIO/qwik/pull/5135